application/src/main/java/org/togetherjava/tjbot/commands/utils/DiscordClientAction.java

```diff
@@ -107,12 +114,16 @@
     /**
      * Contains actions related to the settings menu
      */
-    public static class Settings {
+    @SuppressWarnings("squid:S2972")
```

Comment:
quick comment what it is and why it was disabled please

Reply:
Done

Reply:
nvm, gonna rephrase

Reply:
Thanks. The linter is kinda right. The class isnt really designed nicely with all those constants and "abusing" the classes to enforce some sort of navigation/grouping between them... but I also dont really have any better suggestion right now, other than completely redesigning everything. Im okay with accepting this.

Reply:
Done v2

Reply:
> Thanks. The linter is kinda right. The class isnt really designed nicely with all those constants and "abusing" the classes to enforce some sort of navigation/grouping between them... but I also dont really have any better suggestion right now, other than completely redesigning everything. Im okay with accepting this.

What would your alternative design suggestion be then?

Reply:
I dont have anything concrete to be honest. I struggle finding anything better. I guess a "nice solution" would require to approach this with a completely different mindset.